x/airdrop/keeper/keeper.go

```diff
@@ -19,8 +19,8 @@ type Keeper struct {
 	storeKey         sdk.StoreKey
 	paramSpace       paramtypes.Subspace
 	accountKeeper    authkeeper.AccountKeeper
-	bankKeeper       bankkeeper.Keeper
-	stakingKeeper    stakingkeeper.Keeper
+	BankKeeper       bankkeeper.Keeper
```

Comment:
do these need to be exported? I don't see that they are being used outside of keeper scoped functions.

Reply:
Ah yes, in InitGenesis.